debian/control

```diff
@@ -3,11 +3,10 @@ Section: misc
 Priority: optional
 Maintainer: Andre Meyering <info@andremeyering.de>
 Standards-Version: 4.6.2
-Build-Depends: debhelper (>= 13.0.0),
-               cdbs,
+Build-Depends: debhelper-compat (= 13),
```

Comment:
Thanks. I had issues with debhelper-compat locally, so switched back to debhelper + compat file. 👍

Reply:
If you still want to build for Ubuntu 18, we need to go to debhelper 11. If i recall this correctly it is since debhelper 9 that the compat file can be dropped and `debhelper-compat` can be used as dependency.

Reply:
No, Ubuntu >= 20 is fine. Builds failed for Ubuntu 18 anyway, because of some missing header file (some header has incorrect casing, e.g. `Qmultimedia/` instead of `QMultiMedia/` and I never found the root cause). Also, IIRC, Ubuntu 18 is EOL. If there are still users on Ubuntu 18, they can use the AppImage. :)